app/config/parameters.yml.dist

```diff
@@ -73,3 +73,6 @@ parameters:
 
     # pocket import
     pocket_consumer_key: xxxxxxxx
+    
+    # sites credentials
+    sites_credentials: {}
```

Comment:
I would love to see that in the configuration table instead. Just need to thinking about how can be the best way to handle that.